src/app/login/login.component.ts

```diff
@@ -1,15 +1,31 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core'
+import { FormBuilder, FormGroup } from '@angular/forms'
+import { AuthService } from '../services/auth/auth.service'
 
 @Component({
-  selector: 'tcp-login',
+  selector: 'app-login',
```

Comment:
`tcp-login`  is correct for the selector.  Does the IDE complain about having it set to `app-login`?

Reply:
nope, this was a copy pasta issue, fixed now